homeassistant/components/modbus/sensor.py

```diff
@@ -91,6 +94,7 @@ def number(value: Any) -> Union[int, float]:
                 vol.Optional(CONF_SLAVE): cv.positive_int,
                 vol.Optional(CONF_STRUCTURE): cv.string,
                 vol.Optional(CONF_UNIT_OF_MEASUREMENT): cv.string,
+                vol.Optional(CONF_BIT): cv.positive_int,
```

Comment:
This is change to the old configuration style, I believe this is not allowed.

Reply:
Yes, that's correct. See:
https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md

Reply:
thanks for the link. We are working on changing the whole modbus configuration, but it takes some time, especially since the test harness needs to be partly rewritten.